contracts/Hasher.sol

```diff
@@ -105,4 +105,17 @@ contract Hasher {
 			_account,
 			_nonce);
 	}
+
+	function mapStorageKey(
+		bytes32 _key,
+		uint256 _position)
+		public
+		pure
+		returns (bytes32)
+	{
+		return keccak256(
+			_key,
+			_position);
+	}
+
 }
```

Comment:
Not sure it needs to be in Hasher.sol because it's only used in mock contract i.e. OpenSTValueMock.sol.
May be in HasherMock.sol ?
cc: @pgev

Reply:
Makes sense. Will do!